contracts/protocol/clients/FermionWrapper.sol

```diff
@@ -44,7 +44,11 @@ abstract contract FermionWrapper is FermionFNFTBase, CreatorToken, IFermionWrapp
      *
      */
     constructor(address _seaportWrapper, address _strictAuthorizedTransferSecurityRegistry, address _wrappedNative) {
-        if (_wrappedNative == address(0) || _seaportWrapper == address(0)) revert FermionGeneralErrors.InvalidAddress();
+        if (
+            _wrappedNative == address(0) ||
+            _seaportWrapper == address(0) ||
+            _strictAuthorizedTransferSecurityRegistry == address(0)
```

Comment:
Actually, `address(0)` is a valid value for `_strictAuthorizedTransferSecurityRegistry`.
Even `initializeWrapper` is written in a way to handle it.